allensdk/brain_observatory/behavior/data_objects/stimuli/templates.py

```diff
@@ -30,13 +31,31 @@
 
 class Templates(DataObject, StimulusFileReadableInterface,
                 NwbReadableInterface, NwbWritableInterface):
-    def __init__(self, templates: StimulusTemplate):
+    def __init__(self, templates: Dict[str, StimulusTemplate]):
         super().__init__(name='stimulus_templates', value=templates)
+        # Grab the keys from the input dictionary. The "images" key is assumed
+        # to be the first key in the dictionary that have movie in its key
+        # name. For VBO and VBN releases, there should only be at most 2
+        # keys in the dictionary.
+        image_template_keys = [
+            key for key in templates.keys()
+            if 'movie' not in key.lower()]
+        self._image_template_key = None
+        if len(image_template_keys) > 0:
```

Comment:
can you please raise an exception if there are 2 matching keys? Same for `movie_template_keys`